ui/src/views/Profile/index.jsx

```diff
@@ -45,7 +45,9 @@ export default class Profile extends Component {
                   primary="Certificate"
                   secondary={
                     user.credentials.certificate ? (
-                      <code>{user.credentials.certificate}</code>
+                      <code style={{ wordBreak: 'break-word' }}>
+                        {user.credentials.certificate}
+                      </code>
```

Comment:
Really nice! Could we use the `withStyles` decorator for the styling. Let's separate the jsx from the css. Here's an example: https://github.com/taskcluster/taskcluster/blob/ea9485f2f936ea0c115e26625f6fd2095030216e/ui/src/views/Hooks/ListHooks/index.jsx#L18


```jsx
import { withStyles } from '@material-ui/core/styles';

@withStyles({
  certificate: {
      wordBreak: 'break-word',
  },
})

// ....

<code className={classes.certificate}>
  {user.credentials.certificate}
</code>
```

Reply:
Alright. Got it.

I am just not sure how we can access ```certificate``` using ```classes``` as we never really define it anywhere and we are importing the css with ```withStyles```. What am I missing?

Do I need to define ```classes``` somewhere?

Reply:
The usage of `@withStyles` will inject `classes` in props.

Reply:
Without writing the word ```classes``` explicitly in ```props```, I am getting the error 
> error: 'classes' is not defined (no-undef) at src\views\Profile\index.jsx:54:40:

But after I write `classes` in props myself, everything seems to be working okay.

I will commit the changes for you to review once.